nextflow_aws_batch_trigger.py

```diff
@@ -262,14 +275,30 @@ def main():
     result_location = args.result_location
     secret_id = args.secret_id
     launch_template = {"launchTemplateName": launch_template_name, "version": "$Latest"}
+    instance_type = args.instance_type
+    instance = [args.instance_type]
+    instances = {
```

Comment:
MIssing the other m5 series. Add all. And set this as a global variable in the code.